### ISSUE TITLE
Italicisation for text fields in Janeway that do not currently allow this, e.g. news item titles

### ISSUE THREAD

**Issue by severett89** [openlibhums/janeway#4097]
**Is your feature request related to a problem? Please describe.**
Currently there are certain fields in Janeway that do not support italicisation, such as titles for news items, news item preview text, and certain journal items such as the 'About' text block title, articles/items on the homepage carousel, and the automatic text above the license information on the submissions page.

**Describe the solution you'd like**
Either the ability to add italics with HTML italics tag or where instances of the journal's name is displayed that users cannot edit, that the journal's name displays in italics by default (and that this might be easy to revert if the journal doesn't want this - which is unlikely).

**Additional context**
Journal titles are displayed in italics by the majority of citation styles and so this would not in my view be contributing to the proliferation of styles. The inability to be able to italicise certain titles leads to inconsistency in editorial formatting which is not ideal. This also discourages using the journal's name in places where it should be used, leading to potential accessibility problems (e.g. journal's logo being used as image instead of journal title used as text).

**Comment by ajrbyers** [comment]
Allow HTML in the following places:

- [ ] NewsItem title field
- [ ] NewsItem preview text
- [ ] About homepage element title
- [ ] Carousel text
- [ ] Text above the license block on the submissions page

**Comment by joemull** [comment]
- [ ] These fields also need to be converted to bleach fields if we are marking them as safe.
